api/libproject/devstart.rb

```diff
@@ -10,21 +11,95 @@ def dev_up()
   common.run_inline_swallowing_interrupt %W{docker-compose logs -f api}
 end
 
-def rebuild_image()
+def rebuild_image(args)
   common = Common.new
   common.docker.requires_docker
 
   common.run_inline %W{docker-compose build}
 end
 
+def auth_login(account)
```

Comment:
Good point!

Switched to getting service account credentials and using them with gcloud auth activate-service-account before using gsutil.